backend/configuration/configuration_model.py

```diff
@@ -82,6 +85,11 @@ def __init__(self, local_repo_path, clone_url):
                 # if git repo in local repo path is not the same repo as given in the clone url
                 if not current_clone_url == clone_url:
                     # remove all files form folder and clone new git repo from given clone url
+                    for root, dirs, files in os.walk(self.local_repo_path):
```

Comment:
What was the issue here?

Reply:
This code es needed to remove the ".git" folder in windows. `shutil.rmtree` is not enough to remove read-only files